### ISSUE TITLE
Games suspends on videos

### ISSUE THREAD

**Issue by Solo-Kazuki** [DS-Homebrew/TWiLightMenu#2428]
**Flashcard or Exploits?** 
**Base Console:** 

**TWiLight Menu++ Version:** v27.4.0
**TWiLight Menu++ Theme:** Doesn't matter

### Expected behavior

Video should be played

### Actual behavior

When video should be played game suspends

### Steps to reproduce

Just run until intro or play shortly

### Notes
Games are available here
https://cdromance.org/nds-roms/the-world-ends-with-you-undub/
https://cdromance.org/nds-roms/tales-of-innocence-english-patched/

**Comment by RocketRobz** [comment]
Does the issue happen with the original unhacked ROMs?

**Comment by Solo-Kazuki** [comment]
Yes, You can check it Yourself
https://cdromance.org/nds-roms/tales-of-innocence-japan/
https://cdromance.org/nds-roms/the-world-ends-with-you-europe/
https://cdromance.org/nds-roms/the-world-ends-with-you-usa/

**Comment by RocketRobz** [comment]
@Solo-Kazuki Please don't post links to ROMs.

**Comment by RocketRobz** [comment]
Try turning off card read DMA in the per-game settings menu (opened in TWLMenu++ by pressing Y button).

**Comment by Solo-Kazuki** [comment]
Where to turn it? In "Game Settings" menu I have only Language, Save Number, Game Loader and Bootstrap options.

**Comment by Solo-Kazuki** [comment]
It doesn't work, I have "Start Failed. Error 1" message. Updated as described in guide.
But what about loaders? I must use additionally flashcard loaders which are also updated.

**Comment by RocketRobz** [comment]
> It doesn't work, I have "Start Failed. Error 1" message. Updated as described in guide. But what about loaders? I must use additionally flashcard loaders which are also updated.

See this part of the install guide: https://wiki.ds-homebrew.com/twilightmenu/installing-flashcard#to-run-games-using-your-flashcard-firmware

**Comment by Solo-Kazuki** [comment]
Thank You. Now It works. I didn't install YSMenu that's why it didn't work. And sorry for problems.

**Comment by Solo-Kazuki** [comment]
Ok, Tales of Innocence works, but in kernel mode The World Ends With You have:
[Rom loading failed]
disk errcode=-82

**Comment by Solo-Kazuki** [comment]
Is needed to make new issue regarding The World Ends With You?
It's not working with kernel and with nds-bootstrap game suspends during intro.

**Comment by RocketRobz** [comment]
> Is needed to make new issue regarding The World Ends With You? It's not working with kernel and with nds-bootstrap game suspends during intro.

No. The freezing is likely due to the flashcard you're using. nds-bootstrap is known to work better on Acekard 2(i) or R4(i) Ultra.
